atomicfu-maven-plugin/build.gradle.kts

```diff
@@ -13,31 +19,84 @@ dependencies {
     api(libs.bundles.maven)
 }
 
-val outputDir = tasks.compileKotlin.get().destinationDirectory
+val outputDir = tasks.compileKotlin.get().destinationDirectory.get().asFile
```

Comment:
This looks suspicious to me. Generally we should try to avoid `.get()`, because it will eagerly configure a task. Also, `get().asFile` will make Gradle forget the task link.

Reply:
Fixed